napari/components/viewer_model.py

```diff
@@ -368,6 +378,9 @@ def _on_cursor_position_change(self, event):
         if active is not None:
             self.status = active.get_status(self.cursor.position, world=True)
             self.help = active.help
+            self.tooltip.text = active._get_tooltip_text(
```

Comment:
Can we put this inside `if self.tooltip.visibile:` so that it doesn't get called otherwise

Reply:
done